allensdk/model/biophysical/utils.py

```diff
@@ -71,10 +71,15 @@ def create_utils(description, model_type=None):
         except KeyError as e:
             logging.error("Could not infer model type from description")
 
+    try:
+       axon_type = description.data['biophys'][1]['axon_type']
+    except IndexError or KeyError:
+        axon_type = 'reconstructed_axon'
```

Comment:
it is better to provide this value `reconstructed_axon` as a default when you create an argument with the argparse.